app/views/organizations/index.html.slim

```diff
@@ -17,6 +17,8 @@
           td = organization.owners.map(&:email).join(', ')
           td
             .btn-group.pull-right
+              = link_to 'Switch', switch_organization_path(organization),
+                class: 'btn btn-default btn-sm', id: dom_id(organization), method: :put
```

Comment:
@yunixon use `dom_id` with some prefix here. For example `dom_id(orgranization, :switch)`